bridges/modules/xcm-bridge-hub-router/src/benchmarking.rs

```diff
@@ -47,7 +48,7 @@ benchmarks_instance_pallet! {
 	on_initialize_when_non_congested {
 		Bridge::<T, I>::put(BridgeState {
 			is_congested: false,
-			delivery_fee_factor: MINIMAL_DELIVERY_FEE_FACTOR + MINIMAL_DELIVERY_FEE_FACTOR,
+			delivery_fee_factor: crate::Pallet::<T, I>::MIN_FEE_FACTOR.mul(2.into()),
```

Comment:
Should you use `mul` or `saturating_mul`?

Reply:
Changed it to `saturating_mul`